src/components/Button/SecondaryButton.tsx

```diff
@@ -6,18 +6,14 @@ const SecondaryButton = styled.button<{
   marginTop?: number;
   fontSize?: number;
 }>`
-  font-family: "PTRootUIWebMedium", sans-serif;
+  font-family: 'PTRootUIWebMedium', sans-serif;
   border: none;
   background: transparent;
   font-size: 16px;
   padding: 0;
   ${({ marginTop }) => marginTop && `margin-top: ${marginTop}px`};
   color: ${({ color, theme }) => color || theme.color.text.buttonSecondary};
   cursor: pointer;
-
-  &:hover {
```

Comment:
no longer any indication for hover?

Reply:
Yeah it was mentioned in the ticket, to remove underline on hover